### PR TITLE
♻️ Change amp-story-interactive version to 0.1

### PR DESCRIPTION
**Pull request by mszylkowski**
The components should be in 0.1 instead of 1.0

### CODE REVIEW

File: examples/amp-story/interactive_size.html [hunk 1]
```diff
@@ -0,0 +1,319 @@
+<!--
```
Comment (gmajoulet) [inline examples/amp-story/interactive_size.html:1]: Please remove as it's unrelated to this PR

**Comment by gmajoulet** [comment]
Actually, I can't find where you changed this line in this PR https://github.com/ampproject/amphtml/blob/master/extensions/amp-story-interactive/validator-amp-story-interactive.protoascii#L22

Do you have a validation success test somewhere? I think we can't get to a valid story with the current configuration?

**Comment by kristoferbaxter** [comment]
@gmajoulet Is there a test for the protoascii configuration?

File: examples/amp-story/interactive_size.html (hunk 1 above)
Comment (mszylkowski) [reply examples/amp-story/interactive_size.html]: Oops idk how it got here. Removed

**Comment by gmajoulet** [comment]
I can only see tests for invalid stories, we should also add one valid story. Matias, could you add it? Thanks!

**Comment by mszylkowski** [comment]
Added test that passes, and contains one of each component

File: extensions/amp-story-interactive/0.1/test/validator-amp-story-interactive-valid.html [hunk 2]
```diff
@@ -0,0 +1,89 @@
+<!--
```
Comment (honeybadgerdontcare) [inline extensions/amp-story-interactive/0.1/test/validator-amp-story-interactive-valid.html:1]: Please label which ones are validnad which ones are invalid and why. E.g. before the html segment that tests each condition include `<!-- Valid: amp story interactive poll -->`, `<!-- Invalid: invalid poll due to missing ... -->`

File: extensions/amp-story-interactive/0.1/test/validator-amp-story-interactive-valid.out [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+|
+|    Unless required by applicable law or agreed to in writing, software
+|    distributed under the License is distributed on an "AS-IS" BASIS,
+|    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+|    See the License for the specific language governing permissions and
+|    limitations under the license.
+|  -->
+|  <!--
+|    Test Description:
+|    Correct poll, as well as 2 incorrect polls and a correct binary poll
```
Comment (honeybadgerdontcare) [inline extensions/amp-story-interactive/0.1/test/validator-amp-story-interactive-valid.out:19]: Test description says there are two incorrect polls, but this test passes. Please fix.

Comment (mszylkowski) [reply extensions/amp-story-interactive/0.1/test/validator-amp-story-interactive-valid.out:19]: Good catch, the test is meant to pass so I'll update the description

File: extensions/amp-story-interactive/0.1/test/validator-amp-story-interactive-valid.html (hunk 2 above)
Comment (mszylkowski) [reply extensions/amp-story-interactive/0.1/test/validator-amp-story-interactive-valid.html:1]: In this file all the components are valid (hence the name of the file), and the IDs of the elements are "correct-results" meaning it is valid. On the other validator tests the ID of the elements say what is wrong or right. Eg: missing-endpoint-quiz -> the quiz needs an "endpoint" that was not specified. If the ID is correct-quiz, then it's valid

Comment (honeybadgerdontcare) [reply extensions/amp-story-interactive/0.1/test/validator-amp-story-interactive-valid.html:1]: Please update the test to include these comments for each individual test.

Comment (mszylkowski) [reply extensions/amp-story-interactive/0.1/test/validator-amp-story-interactive-valid.html:1]: Done!

**Comment by mszylkowski** [comment]
@rsimha for presubmit-checks.js owners approval

**Pull request merged**